CRM/Utils/Normalize.php

```diff
@@ -222,6 +222,10 @@ function normalize_address(&$address) {
       'FR' => '/^(\d{5})$/i',
       'NL' => '/^(\d{4})\s*([a-z]{2})$/i',
     );
+    // These will be all-capped
+    $directionals = array(
+      'n', 's', 'e', 'w', 'ne', 'nw', 'se', 'sw'

```

Comment:
You do not need the 1 letter directions here since they will be capitalized with ucfirst()